src/js/utils/__tests__/Util-test.js

```diff
@@ -69,6 +69,14 @@ describe("Util", function() {
   });
 
   describe("#last", function() {
+    describe("with not an array as an input", function() {
```

Comment:
🤔 do we really need this describe block?

Reply:
According to the better spec one should wrap input conditions in a block like:
- with valid input
- without `foo` argument
etc.

Reply:
Wouldn't that mean that you should move the other test into a "with valid input" block and the new one into an "with invalid input" block? You probably should also add at leas another case using an object as input.

Reply:
Yes, but that's gonna be a refactoring already.

Reply:
Shall I also add a string and a number? :)

Reply:
Yes, lets add a number and string as well

Reply:
:sad_troll:

Reply:
✅